ci-operator/config/dora-metrics/pelorus/dora-metrics-pelorus-master__4.13.yaml

```diff
@@ -30,7 +30,9 @@ resources:
       cpu: 500m
       memory: 1Gi
 tests:
-- as: e2e-openshift
+- always_run: false
+  as: e2e-openshift
+  run_if_changed: (^exporters/|^charts/|^pelorus-operator/|^scripts/run-pelorus-e2e-tests.sh$)
```

Comment:
To avoid duplication, can we remove this from `always_run` and `run_if_changed` here? checking here, does not seem necessary https://github.com/openshift/release/blob/master/ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.13.yaml#L69

Reply:
But where would you create that?

The files under `ci-operator/jobs/` are generated so there should be no manual changes to those.